app/models/contact.rb

```diff
@@ -3,6 +3,9 @@
 class Contact < ApplicationRecord
   require 'activerecord-import/base'
   require 'activerecord-import/active_record/adapters/postgresql_adapter'
+  include CleanData
+
+  before_save :strip_whitespace
```

Comment:
Maybe `before_validate` ? in case that affects some validation rule

Reply:
@dmgarland Thats a fair point, I had not thought of that. I will update and put back for review.

Additionally do we want to specify which fields we want to target?  This could be then called with `strip_whitespace <field1> <field2> etc